lib/beaker/host/windows/pkg.rb

```diff
@@ -101,15 +89,7 @@ def pe_puppet_agent_promoted_package_info( puppet_collection = nil, opts = {} )
 
   # @api private
   def identify_windows_architecture
-    arch = nil
-    execute("echo '' | wmic os get osarchitecture", :accept_all_exit_codes => true) do |result|
-      arch = if result.exit_code == 0
-        result.stdout =~ /64/ ? '64' : '32'
-      else
-        identify_windows_architecture_from_os_name_for_win2003
```

Comment:
Should `identify_windows_architecture_from_os_name_for_win2003` be removed as well?